caddyconfig/httpcaddyfile/builtins_test.go

```diff
@@ -173,7 +173,7 @@ func TestRedirDirectiveSyntax(t *testing.T) {
 		},
 		{
 			input: `:8080 {
-				redir /old.html /new.html htlm
+				redir /old.html /new.html html
 			}`,
```

Comment:
We need to check why this test wasn't failing despite a misspelling that's making it an unacceptable value

Reply:
It's deliberate, not a type 🙂 It expects an error, so we're expecting failure and test passes because a legitimate error is returned, as expected. Please revert this one.

Reply:
Done 🍏